esignet-core/src/main/java/io/mosip/esignet/core/config/SharedComponentConfig.java

```diff
@@ -18,4 +19,10 @@ public CaptchaHelper captchaHelper(@Value("${mosip.esignet.captcha.validator-url
                                         @Value("${mosip.esignet.captcha.module-name}") String moduleName) {
         return new CaptchaHelper(restTemplate, validatorUrl, moduleName);
     }
+    
+    @Bean
+    public BCryptPasswordEncoder passwordEncoder() {
```

Comment:
Why is this bean required? there is no description in the ticket to understand the need @balaji-alluru

Reply:
This pr is no longer needed, Thanks